src/DiscordBot.py

```diff
@@ -35,83 +45,84 @@ async def scheduled_task(
     github_utilities: GitHubUtilities, internship_utilities: InternshipUtilities
 ):
     """
-    A scheduled task that runs every 60 seconds to check for new commits in the GitHub repository
-
-    Parameters:
-        - github_utilities: An instance of the GitHubUtilities class
-        - internship_utilities: An instance of the InternshipUtilities class
+    A scheduled task that runs every 60 seconds to check for new commits in the GitHub repository.
     """
+    start_time = datetime.now()
```

Comment:
Change `start_time` to `current_date` and other places as well

Reply:
Will keep `start_time`